package.json

```diff
@@ -38,11 +38,11 @@
     }
   },
   "dependencies": {
+    "@salesforce/bunyan": "^2.0.0",
```

Comment:
@amphro looks like we still run tests for NodeJS 8 but I made changes to bunyan that specify it should be used on NodeJS 10 and above. Which version should be the min version we support here and on bunyan ?

Reply:
We can change it to node10 and node12